scripts/nn/test/grad_check.dml

```diff
@@ -616,6 +617,114 @@ conv2d_simple = function() {
   }
 }
 
+conv2d_transpose = function() {
+  /*
+   * Gradient check for the 2D convolution transpose layer.
+   */
+  print("Grad checking the 2D convolution transpose layer with L2 loss.")
+
+  N = 1
+  C = 1
+  Hin = 3
+  Win = 3
+  F = 1
```

Comment:
Can you increase `N`, `C`, and `F` to `2`?  That will help to ensure that there are no issues related to the shapes that may be masked when those are 1.